troupon/payment/views.py

```diff
@@ -39,7 +39,11 @@ def post(self, request, *args, **kwargs):
                             description=payment_details['description']
                          )
                 # return a success message
-                message = "Success! Your payment has been received."
+                amount = payment_details['amount'] / 100
+                message = "Your payment of $ " + str(amount) + " has been received.\n\
+                Item processing time is a maximum of 10 days.\n \
+                Please contact the supplier for more details."
+
                 messages.add_message(request, messages.WARNING, message)
```

Comment:
**Code Quality: Nitpick**

I thought this was a success message. Not sure why it has the `messages.WARNING` flag.